docs/changelog/v5.md

```diff
@@ -4,6 +4,15 @@ slug: latest
 sidebar_label: Latest (v5)
 sidebar_position: 1
 ---
+# 5.2.3
+- rpc
+  - Add `traverseIterator` method
+  - Change `InvokeResult` to allow an optional specific type for the stack
+- client
```

Comment:
there isnt a standalone `client` folder so i would just remove this since its implied that adding the method to `rpc` would mean the rpc client.